CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## v1.0.0
+
+* Initial release of zync
```

Comment:
@nwt: FWIW since we weren't publishing a changelog before, I was going on the assumption that we could treat this moment at "time zero". But happy to go with your suggestion.